docs/src/content/hardhat-runner/docs/guides/typescript.md

```diff
@@ -128,11 +128,11 @@ For example, if you run `npx hardhat test` and one of your tests has a compilati
 
 Since type-checking adds significant overhead, we recommend to do it only in your CI or in pre-commit/pre-push hooks.
 
-## Writing tests and scripts in TypeScript
```

Comment:
Same in this guide, we shouldn't remove documentation

Reply:
I have reverted some of the script removals in this doc.
I kept the removal of the reference to scripts and the Getting started guide. The Getting started example no longer includes a script as deployment is now through Ignition